helpers/lib/cdnify.js

```diff
@@ -76,6 +78,8 @@ module.exports = globals => {
             return [cdnUrl, 'stencil', versionId, 'e', editSessionId, path].join('/');
         }
 
-        return [cdnUrl, 'stencil', versionId, path].join('/');
+        urlElements = [cdnUrl, 'stencil', versionId, path].map((element, idx) => { return (element[0] === '/' || idx === 0) ?  element : "/" + element });
+        
+        return urlElements.join("");
```

Comment:
This doesn't solve all return paths from this function.

I think the solution to this issue is much easier. Before calling `join('/')`, we must ensure that `path` does not contain a leading `/`. You can see that we are explicitly adding that above. Instead of adding the leading `/`, we should remove it.

Reply:
@mattolson  This leading / is required to the check of the check of the assets: https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/lib/cdnify.js#L71
If we not adding / in places that this one is exist the returned path is correct